### PR TITLE
moved spectrogram2 into spectrogram and removed all old code

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: docs/overview.rst [hunk 1]
```diff
@@ -2,108 +2,11 @@
...
-    interesting = nobg.in_interval("06:27")
-    interesting.peek(vmin=20)
-
-If we want more context, we can also join together different images into a large one in time (note that this does more than just concatenating the array and the axes -- it also considers possible overlap or gaps):
-
-.. code-block:: python
-
-    c1 = CallistoSpectrogram.read('BIR_20110922_101500_01.fit')  # doctest: +SKIP
-    c2 = CallistoSpectrogram.read('BIR_20110922_103000_01.fit')  # doctest: +SKIP
-    d = CallistoSpectrogram.join_many([c1, c2])  # doctest: +SKIP
```
Comment (nabobalis) [inline docs/overview.rst:103]: Missing from Spectrogram now

File: docs/overview.rst [hunk 2]
```diff
@@ -2,108 +2,11 @@
...
-    :include-source:
-
-    import matplotlib.pyplot as plt
-    import sunpy
-    import sunpy.data.sample
-
-    from radiospectra.sources.callisto import CallistoSpectrogram
-    image = CallistoSpectrogram.read(sunpy.data.sample.CALLISTO_SPECTRUM)
-    nobg = image.subtract_bg()
-    interesting = nobg.in_interval("06:27")
```
Comment (nabobalis) [inline docs/overview.rst:94]: Missing from Spectrogram now

File: docs/overview.rst [hunk 3]
```diff
@@ -2,108 +2,11 @@
...
-
-.. plot::
-    :include-source:
-
-    from matplotlib import pyplot as plt
-    import sunpy.data.sample
-    from radiospectra.sources.callisto import CallistoSpectrogram
-
-    image = CallistoSpectrogram.read(sunpy.data.sample.CALLISTO_SPECTRUM)
-    bg = image.auto_const_bg()
```
Comment (nabobalis) [inline docs/overview.rst:60]: Missing from Spectrogram now

File: docs/overview.rst [hunk 4]
```diff
@@ -2,108 +2,11 @@
...
-.. plot::
-    :include-source:
-
-    from matplotlib import pyplot as plt
-    import sunpy
-    import sunpy.data.sample
-    from radiospectra.sources.callisto import CallistoSpectrogram
-
-    image = CallistoSpectrogram.read(sunpy.data.sample.CALLISTO_SPECTRUM)
-    nobg = image.subtract_bg()
```
Comment (nabobalis) [inline docs/overview.rst:47]: Missing from Spectrogram now

**Comment by samaloney** [comment]
This would make things a bit easier, as long it really explicit in the release notes (which are very bare see  #81 )

**Pull request merged**